src/providers/v2v3/V2V3PayProjectFormProvider.tsx

```diff
@@ -1,31 +1,16 @@
 import { PayProjectFormContext } from 'components/Project/PayProjectForm/payProjectFormContext'
 import { usePayProjectForm } from 'components/Project/PayProjectForm/usePayProjectForm'
 import { V2V3PayButton } from 'components/v2v3/V2V3Project/V2V3PayButton/V2V3PayButton'
-import { NftRewardsContext } from 'contexts/nftRewardsContext'
 import { V2V3ProjectContext } from 'contexts/v2v3/V2V3ProjectContext'
 import { useContext } from 'react'
-import { getNftRewardTier } from 'utils/nftRewards'
 import { weightAmountPermyriad } from 'utils/v2v3/math'
 
 export const V2V3PayProjectFormProvider: React.FC = ({ children }) => {
   const { fundingCycleMetadata, fundingCycle, tokenSymbol, tokenAddress } =
     useContext(V2V3ProjectContext)
-  const {
-    nftRewards: { rewardTiers: nftRewardTiers },
-  } = useContext(NftRewardsContext)
 
   const payProjectForm = usePayProjectForm()
 
-  const isEligibleForNft =
```

Comment:
Don't need this anymore because we're adding selected NFTs to `payMetadata` so we can just rely on it.

Reply:
nice